scss/civicrm/administer/common/_mixins.scss

```diff
@@ -16,8 +16,16 @@
         color: $gray-dark;
```

Comment:
why is the mixing called `civihr_table` if this is used to style core screens?

Reply:
@AkA84 This is from year ago and it seems the reason would be because initially it was only for civihr.